conda-incubator/conda-store-ui
# Bye tabs, hi URLs

Closes #386

### Testing

The automatic tests should give some confidence that this PR does not horribly break anything. That said, a manual run-through should be done before approving and merging.

Types of website actions affected by this PR that should be tested manually in various combinations:

- Open an environment
- Create a new environment
- Go back
- Go forward
- Click edit
- Click save
- Click cancel
- Click delete
